tests/tests/datm_ocn_ice_mx025_cfsr

```diff
@@ -0,0 +1,63 @@
+#
+#  CPLD_DATM_MOM6_CICE_CMEPS_1_STEP_COLD test
+#
+
+export TEST_DESCR="DATM-MOM6-CICE-CMEPS - 1-step cold start"
+
+export CNTL_DIR="datm_ocn_ice_mx025_cfsr"
+export CNTLMED_DIR="datm_ocn_ice_mx025_cfsr/RESTART"
+
+export LIST_FILES="RESTART/MOM.res.nc \
+                   RESTART/MOM.res_1.nc \
+                   RESTART/MOM.res_2.nc \
+                   RESTART/MOM.res_3.nc \
+                   RESTART/iced.2011-10-02-00000.nc \
+                   RESTART/DATM_CFSR.cpl.r.2011-10-02-00000.nc \
+                   ocn_2011_10_01_21.nc"
```

Comment:
Why are we comparing the ocn output file? Usually we compare only the restarts.

Reply:
Ocean output is removed from baseline.

Reply:
Thanks for removing the ocean output. The FV3 output we compare for the coupled tests is instantaneous but the MOM6 output is averaged so I don't think we want to compare it.

Currently for the coupled model we're not comparing the mediator files because sometimes they fail comparison even when they are identical when compared manually. Do we know for sure that we can safely compare the mediator files for the DATM and not have false negatives?

Reply:
Yes, we can safely compare the mediator files.